org.eclipse.wildwebdeveloper/pom.xml

```diff
@@ -93,9 +93,18 @@
 							<skipCache>true</skipCache>
 							<url>https://raw.githubusercontent.com/microsoft/vscode-eslint/release/2.2.2/server/package.json</url>
 							<outputDirectory>${project.build.directory}/vscode-eslint-ls/extension/server</outputDirectory>
-							<headers>
-								<Authorization>${github.api.token}</Authorization>
-							</headers>
+							<if>
+								<condition>
+									<not>
+										<equals arg1="${github.api.token}" arg2="" />
+									</not>
+								</condition>
+								<then>
+									<headers>
+										<Authorization>token ${github.api.token}</Authorization>
+									</headers>
+								</then>
+							</if>
```

Comment:
All this is ignored. Please use Maven profiles for conditional configuration.

Reply:
@mickaelistria  I've changed the PR to use a profile. Could you please review?

Also, please note that in last build (https://ci.eclipse.org/wildwebdeveloper/job/Wildwebdeveloper/job/PR-822/6/) I've switched it back to your image (JDK 11, Node.js v16.13.0) in order to see if it fixes the JUnit test failures (not successful though), so it probably worth change it again to use OpenJ9 JDK 17 + Node.js 16.14.2 before we merge it (if we're OK to do so)